Sylvance/mwavuli
# ch Continuous integration

What does this PR do?
Adds continuous integration.

Description of Task to be completed?
Add Travis, Coveralls and Codacy.

How should this be manually tested?
Do a git commit and push. Check out coveralls and travis.

Any background context you want to provide?
This is the start of CI.

Screenshots (if appropriate)
Questions:
